pkg/cli/cli.go

```diff
@@ -102,7 +102,7 @@ func RunParser(options Options) error {
 	}
 	httpClient.AuthToken = githubAPIToken
 
-	suiteCategories, err := github.CollectSuiteCategories(repoConfig, httpClient)
+	suiteCategories, err := github.CollectSuiteCategories(repoConfig, httpClient, options.Version)
```

Comment:
This logic has no associated test when version is not `Unreleased` afaict

Reply:
it's in the github package in `TestCollectSuiteCategories` - `2 version diff between suite versions, using release branch with full changelog`

the first two TCs use a blank value for the version (which isn't the default "unreleased", but it's fine). the 3rd case uses `real_release_branch` as the version which hits a different changelog file `release_branch_changelog.md`, and that file has an additional tag that `simple_changelog.md` doesn't have. this mimics the real life situation we sometimes have, where release branches have tags / changelog updates that are not available on the main/master branch.